### PR TITLE
Spike/cloudinary GitHub actions

### PR DESCRIPTION
**Pull request by delete-44**
- [x] Adds GH action to read all files in the `Inscryption/Uploads` folder on cloudinary that are > 1 day old, and delete them
- [x] To test, visit [this action](https://github.com/delete-44/inscryber/actions/workflows/cloudinary_cleanup.yml) and click "Run workflow" from `spike/....`
- You may need to refresh page to see your new workflow
image

---

Edit:
Also, an example of it actually working: https://github.com/delete-44/inscryber/runs/5204443886?check_suite_focus=true

### CODE REVIEW

**Review by ReeceDrage — APPROVED** [review]
I can't see anything obvious that's been missed here 😃

File: .github/workflows/cloudinary_cleanup.yml [hunk 1]
```diff
@@ -1,18 +1,58 @@
...
+      # Output of this response returns full metadata for each image. We don't need
+      # most of it, so use command line tool JQ to build an array of public IDs.
+      # This is stored as an output so we can use in later steps
+      - name: Filter output with jq
+        id: jq-filter
+        run: echo ::set-output name=RESOURCES::$(echo ${{ toJSON(steps.get-request.outputs.response) }} | jq '.resources | [.[].public_id]')
+
+      - name: LOG all IDs to be deleted
+        run: echo ${{ steps.jq-filter.outputs.RESOURCES }}
+
```
Comment (nodonnell98) [inline .github/workflows/cloudinary_cleanup.yml:40]: not a change but a question, are these logs just futureproofing for debugging later down the line? If so - nice

**Review by nodonnell98 — APPROVED** [review]
looks great, kudos :shipit:

File: .github/workflows/cloudinary_cleanup.yml (hunk 1 above)
Comment (delete-44) [reply .github/workflows/cloudinary_cleanup.yml:40]: Absolutely this, yeah. The only way to debug anything in GH actions is to `echo` it (afaik) and I *know* that six months down the line this is going to break and I'm going to forget how to access any of the variables :P

**Pull request merged**